docker/versions.dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG ANDROID_CMD_VERSION=11076708
+ARG ANDROID_CMD_VERSION=13114758
```

Comment:
I've checked, and this version still supports Java 17:

```
cmdline-tools/lib/sdklib/libsdkmanager_lib/com/android/sdklib/tool/sdkmanager/SdkManagerCli.class: compiled Java class data, version 61.0 (Java SE 17)
```

So let's just merge this.